### ISSUE TITLE
Is the BERT QA pre-trained model with sklearn wrapper fine-tuned with new whole word masking?

### ISSUE THREAD

**Issue by edanweis** [cdqa-suite/cdQA#240]
Were the cdQA pre-trained models made available after BERT released the new whole word masked versions back in May? Apparently they resulted in improved performance ~93 F1 on SQuAD 2.0

Also, why wait for XLNet PR before making available BERT QA models fine-tuned on SQuAD 2.0, as @fmikaelian mentioned in [#157](https://github.com/cdqa-suite/cdQA/issues/157#issuecomment-513453504)?

Would it be possible to convert a TF checkpoint trained on SQuAD 2.0 and then create a sklearn wrapper .joblib file? How would I do that?

Keep up the great work people. :)

**Comment by andrelmfarias** [comment]
Hi @edanweis, thanks for these great questions! Here are my answers:

1) No they weren't as they were trained before the release of the [v1.0.0](https://github.com/huggingface/pytorch-transformers/releases/tag/v1.0.0) of HF transformers repo. It was their first release where they included the whole word masked version. We might train the models again using the last version of models released by HF

2) The only reason is that we wanted to optimize our work by applying these two tasks to a unique PR. But actually, it is maybe better to train BERT on SQUAD 2.0 without having to wait for the integration of XLNet.

3) The easiest way to do that, IMO, it to convert your TF checkpoint to a PyTorch version of HuggingFace's BERT for QA. There's an article on Medium published by Thomas Wolf, the Scientist Lead at Hugging Face explaining how you can do that: https://medium.com/huggingface/from-tensorflow-to-pytorch-265f40ef2a28
Then you would only need to load this model when creating an instance of the classes `BertQA` or `QAPipeline`.

Also if you are willing to and think you can help with some of these tasks, you are more than welcome 😃

**Comment by edanweis** [comment]
Thanks @andrelmfarias. I stumbled across this [issue; #1050](https://github.com/huggingface/pytorch-transformers/issues/1050) - which led me to [convert_tf_checkpoint_to_pytorch.py](url) bt huggingface. Could this be an easier way to convert? 

Am I able to convert fine-tuned models using this method too?

**Comment by andrelmfarias** [comment]
I think you can use this script. I have never tried, but it seems to do the job. By reading quickly the code I understand that the script does not use the architecture of the TF model but the weights as well, so you can use it for fine-tuned models.

**Comment by andrelmfarias** [comment]
Closing this issue due to inactivity
